arangod/Indexes/Index.h

```diff
@@ -113,6 +113,17 @@ class Index {
   /// @brief return the index id
   inline TRI_idx_iid_t id() const { return _iid; }
 
+  /// @brief return the index name
+  inline std::string const& name() const {
+    if (_name == StaticStrings::IndexNameEdgeFrom || _name == StaticStrings::IndexNameEdgeTo) {
+      return StaticStrings::IndexNameEdge;
```

Comment:
Is this going to cause problems if two indexes have the same name, e.g. in the RocksDB engine?
Additionally, what happens if a user creates an index with name `StaticStrings::IndexNameEdge` in a non-edge collection?

Reply:
And how does one distinguish between the `_from` part of the `_to` part in the RocksDB engine when running a query and using the "force index" feature? If both edge indexes pretend to have the same name, as a user there is no way to select the "right" one of the two, right?